client/components/ImportModule.js

```diff
@@ -19,12 +19,17 @@ export default class ImportModule extends React.Component {
             let asJSON;
             try {
                 asJSON = JSON.parse(asText);
+
+                Actions.ImportInstance({
+                    content: asJSON
+                });
             } catch(e) {
-                alert('JSON file to import isnt valid!');
+                if (e instanceof SyntaxError) {
+                    alert('JSON file to import isnt valid!');
+                } else {
+                    alert(['Unexpected error', ex.message].join(':'));
```

Comment:
This is ES6! :) You can use template strings here:

     alert(`Unexpected error: ${ex.message}`);